Lib/shutil.py

```diff
@@ -297,15 +297,16 @@ def _copyxattr(src, dst, *, follow_symlinks=True):
         try:
             names = os.listxattr(src, follow_symlinks=follow_symlinks)
         except OSError as e:
-            if e.errno not in (errno.ENOTSUP, errno.ENODATA):
+            if e.errno not in {errno.ENOTSUP, errno.ENODATA, errno.EINVAL}:
```

Comment:
There is no benefit from using a set instead of a tuple. Building a set is more expensive than builtin a tuple.
```suggestion
            if e.errno not in (errno.ENOTSUP, errno.ENODATA, errno.EINVAL):
```